src/Gui/PreferencePackManager.cpp

```diff
@@ -224,7 +224,7 @@ void Gui::PreferencePackManager::importConfig(const std::string& packName,
     auto savedPreferencePacksDirectory =
         fs::path(App::Application::getUserAppDataDir()) / "SavedPreferencePacks";
     auto cfgFilename = savedPreferencePacksDirectory / packName / (packName + ".cfg");
-    fs::copy_file(path, cfgFilename, fs::copy_option::overwrite_if_exists);
+    fs::copy_file(path, cfgFilename, fs::copy_options::overwrite_existing);
```

Comment:
I guess that's more related to the used boost version that deprecates `overwrite_if_exists` and not so much to the user compiler version. At least this change causes a build failure for Ubuntu 20.04 and Windows Conda.

So, this should be handled with an `ifdef` depending on the used boost version

Reply:
sure.
according to [the commit](https://github.com/boostorg/filesystem/commit/f199152b7df036ff1606c85e4ea1b28edfeda6cc)
boost add the deprecation warning since 1.74